### ISSUE TITLE
TextArea | ensure every form element has a label

### ISSUE THREAD

**Issue by dexterca** [lifeomic/chroma-react#97]
Title: WCAG 1.3.1,WCAG 4.1.2: Ensures every form element has a label (#textArea-a7f1824c85aaa71f36e2a4b784193ed9)
Tags: Accessibility, WCAG 1.3.1, WCAG 4.1.2, label

Issue: Ensures every form element has a label (label - https://accessibilityinsights.io/info-examples/web/label)

Target application: Form Components / TextArea - Default ⋅ Storybook - http://localhost:9001/?path=/story/form-components-textarea--default

Element path: #storybook-preview-iframe;#textArea-a7f1824c85aaa71f36e2a4b784193ed9

Snippet: <textarea aria-describedby="error-for-textArea-a7f1824c85aaa71f36e2a4b784193ed9" class="ChromaTextArea-textarea-90 ChromaTextArea-textareaInverse-91 ChromaTextArea-textareaError-92" cols="1" id="textArea-a7f1824c85aaa71f36e2a4b784193ed9"></textarea>

How to fix: 
Fix all of the following:
  Form element has explicit <label> that is hidden

Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  Element has no title attribute
  Element has no placeholder attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

Environment: Chrome version 89.0.4389.82

====

This accessibility issue was found using Accessibility Insights for Web 2.24.0 (axe-core 4.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
